src/main/target/common_post.h

```diff
@@ -319,6 +319,10 @@
 #undef USE_RANGEFINDER_TF
 #endif
 
+#if (defined(ACC_1_ALIGN) || defined(ACC_2_ALIGN))
```

Comment:
If we do this, we should be doing it consistently, and check for the 50 or so other deprecated defines that are used in target definitions as well. We should probably move it into `common_deprecated_post.h`.

Reply:
That makes sense. Do we have a list or know a number of them to start with? Probably makes sense to do this in another PR though.

Reply:
I looked into tackling this from the other end, and extract a list of all used defines in `target.h` files, and then fail the test if any of these were not used in an `#ifdef`. Back then it was literally 100s of defines to be removed. I could try running this again, and see if the list has shrunk.

Reply:
I think that maybe this was a separate analysis. Looking at unused defines is just a cleanup task mostly. For things that are truly deprecated (previously valid and used but now no longer needed) is a separate analysis that will probably require looking at all the current defines in the `target.h` and comparing them to the same in previous versions. This would at least give a starting candidate list for examination. But I guess that even this isn't perfect because it will get fooled by any targets in master that erroneously still have the deprecated defines. So maybe we just start with the few knowns like `ACC_x_ALIGN`, `USE_MULTI_GYRO`, `USE_DUAL_GYRO`, etc. and add as we identify others.